vtex-apps/klaviyo-pixel
# [KLAVIYO-12] Update the started checkout code in the readme file

###Motivation
After June 13th, our Identify API will no longer support making updates to an existing profile’s unique identifiers (email address, phone number, and external id). Now, to make these profile  updates, you will need to utilize Klaviyo’s Profiles API, which requires using an account’s Private API Key.

### How to test it
You can use [this workspace to test](https://arturovtex--horzeusa.myvtex.com)
and test the next new routes:
`/klaviyo/profile/:email`
`/klaviyo/updateProfile/:profileId`

These routes should be used only in the checkout code to send the profile info to Klaviyo during the checkout.
